### PR TITLE
Create .editorconfig

### PR DESCRIPTION
**Pull request by koenpunt**
As suggested by @MasonM; (https://github.com/jpfuentes2/php-activerecord/pull/532/files/03aff5853c171c3d2baf469a6dcde1795656369f#r58916886)

Have anything to add to this?

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
```
Comment (MasonM) [inline .editorconfig]: Should be "tab". The only file that uses spaces for indentation appears to be lib/Utils.php (it actually mixes the two).

Comment (MasonM) [inline .editorconfig]: whoops, missed the `[*.php]` line that overrides this. Nevermind!

Comment (koenpunt) [inline .editorconfig]: The only filetype that uses tabs is php, which is defined below ;) json, xml, yml and md all use spaces.

**Comment by MasonM** [comment]
Looks good! You could fix everything that deviates from this using [editorconfig-tools](https://www.npmjs.com/package/editorconfig-tools). Commands:

```
npm install editorconfig-tools
find . -name "*php" -not -path "*/vendor/*" | xargs ./node_modules/.bin/editorconfig-tools fix
```

**Comment by koenpunt** [comment]
Even though that's nice, there are a lot of other code inconsistencies that have to be fixed, so for now I think adding this config is a good first start.

**Pull request merged**